src/app/modules/main/profile_section/profile/models/profile_preferences_communities_model.nim

```diff
@@ -184,16 +173,25 @@ QtObject:
     if ind != -1:
       self.remove(ind)
 
-  proc move*(self: ProfileShowcaseCommunitiesModel, fromIndex: int, toIndex: int) {.slot.} =
-    if fromIndex < 0 or fromIndex >= self.items.len:
+  proc move*(self: ProfileShowcaseCommunitiesModel, fromRow: int, toRow: int, dummyCount: int = 1) {.slot.} =
```

Comment:
`dummyCount` will probably end up in a warning because it is not used. Try to use `_`

Reply:
Ah, honestly didn't notice as there are too many warning already :/